### ISSUE TITLE
Considering creating a merged guideline called Non-Keyboard Accessible

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag21#852]
Please see the [latter part of the discussion in Issue 806](https://github.com/w3c/wcag21/issues/806#issuecomment-373720799) for background.

The key issue is that the **Pointer Accessible** and **Additional Sensor Inputs** guidelines are not consistent  (How is Concurrent Input Mechanisms only about Pointer?) and not well-named to accommodate  several 'orphan' SCs in other guidelines ( How is Label in Name only about Navigable? Where can it go?) 

We're suggesting taking the Pointer Accessible and Additional Sensor Inputs guidelines and merging them and all their SCs into one renamed Guideline called **Non-Keyboard Accessible**, then moving over a few orphans there. That gets us to a more comfortable fit, and reduces future (2.2 and beyond) contortions. The changes proposed are asterisked.

***2.5 Non-Keyboard Accessible** [name change]
2.5.1 Pointer Gestures
2.5.2 Pointer Cancellation
2.5.3 Target Size
*2.5.4 Motion Actuation
*2.5.5 Orientation
  *2.5.6 Label in Name
  *2.5.7 Concurrent Input Mechanisms
  *2.5.8 Character Key Shortcuts [see below]

I'd also recommend moving Character Key Shortcuts out of this Guideline and into
**2.1 Keyboard Accessible**
*2.1.4 Character Key Shortcuts

**Comment by mbgower** [comment]
Note that another idea was moving Label in Name to robust (just as Status Messages was).

**Comment by Ryladog** [comment]
Thanks for putting this together again Mike. It is important that our Guidelines be descriptive of the SC below them. Having a Guideline that says 'Additional' or 'Other' is not descriptive. I like Non-Keyboard Accessible for this GL. I also agree with moving Character Key Shortcuts below Guideline 2.1 Keyboard Accessible. @nitedog

**Comment by Ryladog** [comment]
I recall that there was no time left to discuss this part of this topic,
after you have gotten the GL name change for you wanted for Seizures and
moving the Animations from Interactions SC, - and - that you said the WG
could discuss the concerns of Mike and I on email after the F2F. I remember
this clearly.   @nitedog <https://github.com/nitedog>

**Comment by Ryladog** [comment]
I incompletely agree with Mike Gower on this.

** katie **

*Katie Haritos-Shea*
*Principal ICT Accessibility Architect *

*WCAG/Section 508/ADA/AODA/QA/FinServ/FinTech/Privacy,* *IAAP CPACC+WAS = *
*CPWA* <http://www.accessibilityassociation.org/cpwacertificants>

*Cell: **703-371-5545 <703-371-5545>** |* *ryladog@gmail.com
<ryladog@gmail.com>* *| **Oakton, VA **|* *LinkedIn Profile
<http://www.linkedin.com/in/katieharitosshea/>*

People may forget exactly what it was that you said or did,
but people will never forget how you made them feel.......

Our scars remind us of where we have been........they do not have to
dictate where we are going.

**Comment by awkawk** [comment]
> I incompletely agree with Mike Gower on this.

I'm pretty sure I incompletely agree with him also!

**Comment by Ryladog** [comment]
:-)

** katie **

**Comment by nitedog** [comment]
If it helps this discussion, @kwahlbin had suggested the following structure, which we used in our CSUN presentation (it is only a partial structure for the mobile SCs):

# Speech Input
- Character Key Shortcuts
- Label in Name

# Pointer
- Pointer Gestures
- Pointer Cancellation
- Target Size

# Input Methods
- Concurrent Input Mechanisms
- Motion Actuation

# Device Settings
- Orientation

# Status Changes

**Comment by mraccess77** [comment]
Personally I think the term "non-keyboard accessible" is confusing and will ultimately be problematic.  Some might assume it means that items under it don't need to be keyboard accessible.

**Comment by Ryladog** [comment]
I like Kathy's suggestions over Non-keyboard as well

**Comment by lauracarlson** [comment]
@mraccess77 wrote:

>Personally I think the term "non-keyboard accessible" is confusing and will ultimately be problematic. Some might assume it means that items under it don't need to be keyboard accessible.

Agreed.

**Comment by goodwitch** [comment]
Seems to me that all of these should really be under a heading called "Input Methods".

* Input Methods
  * Speech Input
    * Character Key Shortcuts
    * Label in Name
  * Pointer Input
    * Pointer Gestures
    * Pointer Cancellation
    * Target Size
  * Other Input Methods
    * Concurrent Input Mechanisms
    * Motion Actuation
  * Device Settings
    * Orientation

And...then...keyboard stuff should be under "input methods" too...in my humble opinion. 

I love the idea of doing this...but I don't want to slow down moving to PR.  If this is just editorial...I feel we could do it at any time.

**Comment by mbgower** [comment]
> Seems to me that all of these should really be under a heading called "Input Methods".

And funnily enough, that was the destination Katie and I arrived at in our review, which I just sent to the group a short while ago.  The key difference is that we didn't put _everything_ under there, and we just left it as a guideline rather than make it a Principle, as your hierarchy might suggest. I also made the argument that things will often apply to more than one input, so trying to compartmentalize things too much will lead to headaches. That is why we renamed Pointer Accessible as Input Methods; the latter can include the former.

> And...then...keyboard stuff should be under "input methods" too...in my humble opinion

Yes, we also acknowledged that -- although Keyboard Accessible is part of 2.0 and the change would have a significant impact on backward compatibility, so is a no-fly zone. We felt keyboard is fairly well defined as a baseline, and Input Methods helps us tackle issues that specifically deal with stuff beyond that keyboard baseline.

**Comment by alastc** [comment]
I think this has been passed by events, can we close it?

**Comment by awkawk** [comment]
Yes, should close.
